projects/igniteui-angular/src/lib/grids/grid/grid.component.ts

```diff
@@ -238,6 +238,11 @@ export class IgxGridComponent extends IgxGridBaseComponent implements IGridDataB
      *     dir: SortingDirection.Asc,
      *     ignoreCase: false
      * }];
+     * ```
+     *
+     * Two-way data binding.
+     * ```html
+     * <igx-grid #grid [data]="Data" [autoGenerate]="true" [(groupingExpressions)]="model.groupingExpressions"></igx-grid>
```

Comment:
@MayaKirova Setting both `groupingExpressions` and `groupingExpansionState` two-way data binding throws 'Maximum call stack size exceeded' error.